sub-packages/bionemo-geneformer/src/bionemo/geneformer/run/main.py

```diff
@@ -93,7 +94,7 @@ def string_to_class(path: str):
 
     def load_config(config_path: str, model_config_t: Optional[str], data_config_t: Optional[str]) -> MainConfig:
         with open(config_path, "r") as f:
-            config_dict = json.load(f)
+            config_dict = yaml.safe_load(f)
```

Comment:
Not revew-blocking: you could consider _adding_ YAML support instead of replacing the existing JSON support w/ YAML support only. For example, something like:
```
try:
  with open(config_path, 'rt') as rt:
      config_dict = yaml.safe_load(rt)
except <yaml format exception> as error_yaml:
   try:
       with open(config_path, 'rt') as rt:
         config_dict = json.load(rt)
   except JSONDecodeError as error_json:
         try:
             raise ValueError("YAML parsing and JSON-parsing fallback failed for {config_path=}") from error_json
         except Exception as e:
             raise e from error_yaml
```

Reply:
imho, supporting many config formats might introduce some ambiguity, as it would require us to maintain two different configuration formats while our codebase and CLI are still evolving significantly. However, it could be worth considering as a potential feature in the future.

Reply:
I agree with dorota, we should keep it with one way to do things IMO.